### PR TITLE
[Enhanced] Add region-lock to sign-up form

### PR DESCRIPTION
**Pull request by francoislehoux-okta**
## Description:
- **What's changed?** Add region-lock to sign-up form (RU and BY)

### Resolves:

* [OKTA-480603](https://oktainc.atlassian.net/browse/OKTA-480603)

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/util/geoLocation.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+const STORAGE_KEY = 'oktaLocation';
+const ONE_WEEK_IN_MILLISECONDS = 1000 * 60 * 60 * 24 * 7;
+
+class GeoLocation {
+  constructor(onCompletion) {
+    this.hasRetried = false;
+
+    if (typeof onCompletion === 'function') {
+      this.onCompletion = onCompletion;
+    }
```
Comment (shawyu-okta) [inline packages/@okta/vuepress-theme-prose/util/geoLocation.js:14]: nit: could write this like
```js
this.onCompletion = typeof onCompletion === 'function' ? onCompletion : () => {};
```
then you don't need to guard the calls later on

Comment (shawyu-okta) [reply packages/@okta/vuepress-theme-prose/util/geoLocation.js:14]: or you can use `?.` as a guard

**Review by shawyu-okta — APPROVED** [review]

File: packages/@okta/vuepress-theme-prose/util/geoLocation.js (hunk 1 above)
Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/util/geoLocation.js:14]: Even better: `constructor(onCompletion = () => {})`

Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/util/geoLocation.js:14]: Nothing enforces that `onCompletion` is a function though

**Pull request merged**